connector/client/client.go

```diff
@@ -87,6 +89,24 @@ func (client *ConnectorClient) Delete(target url.URL, activity schema.Activity)
 	return client.sendRequest(req, activity)
 }
 
+// Put an activity.
+//
+// Creates a HTTP PUT request with the provided activity payload and a Bearer token in the header.
+// Returns any error as received from the call to connector service.
+func (client *ConnectorClient) Put(target url.URL, activity schema.Activity) error {
+	jsonStr, err := json.Marshal(activity)
+	if err != nil {
+		return err
+	}
+	fmt.Println(target.String())
```

Comment:
Debug statement? Can we remove it or log it in debug mode?

Reply:
Sure will move it to debug.